Vendor/CrashReporter.framework/Versions/A/Headers/PLCrashNamespace.h

```diff
@@ -35,25 +35,16 @@
  * This may be used to avoid symbol conflicts between multiple libraries
  * that may both incorporate PLCrashReporter.
  */
- #define PLCRASHREPORTER_PREFIX Rollbar
+// #define PLCRASHREPORTER_PREFIX AcmeCo
```

Comment:
I thought we needed to compile PLCrashReporter ourselves for this to actually work?